#!/bin/sh /usr/share/dpatch/dpatch-run
## 02-main.dpatch by Daniel Baumann <daniel.baumann@panthera-systems.net>
##
## DP: Updates URL to new site

@DPATCH@

diff -Naur tinyerp-client-3.1.0.orig/bin/modules/gui/main.py tinyerp-client-3.1.0/bin/modules/gui/main.py
--- tinyerp-client-3.1.0.orig/bin/modules/gui/main.py	2005-10-07 07:01:26.000000000 +0000
+++ tinyerp-client-3.1.0/bin/modules/gui/main.py	2005-10-16 13:12:13.550565320 +0000
@@ -281,7 +281,7 @@
 		about.signal_connect("on_but_ok_pressed", lambda obj: about.get_widget('win_licence').destroy() )
 
 	def sig_webhelp(self, widget):
-		webbrowser.open('http://tinyerp.org/docs/index.php/HomePage.html')
+		webbrowser.open('http://tinyerp.org/wiki/index.php/Main/HomePage')
 
 	def sig_webcontext(self, widget):
 		model = self._wid_get().model
